src/main/java/de/nadirhelix/guestbook/image/facade/impl/ImageFacadeImpl.java

```diff
@@ -15,10 +15,11 @@
 import org.springframework.stereotype.Component;
 
 import de.nadirhelix.guestbook.image.facade.ImageFacade;
+import de.nadirhelix.guestbook.util.PathFinder;
 
 @Component("imageFacade")
 public class ImageFacadeImpl implements ImageFacade {
-
+	
 	private ResourceLoader resourceLoader;
 
     @Autowired
```

Comment:
Ah, das brauchst du so nicht unbedingt machen. Du kannst das auch ohne Konstruktor mit <at>Resource einfach am ResourceLoader.
Es gibt viele Wege die Instanzen zu injizieren und das ist die Variante über einem Konstruktor. Solange es nicht absolut notwendig ist, würde ich bei der Variante ohne Konstruktor bleiben und nur die Felder der Klasse annotieren. Oder ist es in diesem Fall nötig gewesen?

Reply:
Siehe Kommentar in ResourceFacadeImpl :-)